tests/testthat/test-csv.R

```diff
@@ -3,9 +3,9 @@ context("read_cmdstan_csv")
 if (not_on_cran()) {
   set_cmdstan_path()
   fit_bernoulli_optimize <- testing_fit("bernoulli", method = "optimize")
-  fit_bernoulli_variational <- testing_fit("bernoulli", method = "variational")
+  fit_bernoulli_variational <- testing_fit("bernoulli", method = "variational", seed = 123)
```

Comment:
This has nothing to do with this PR. Ever since I added these, some tests kept failing randomly. It seems that variational fails with some seeds for the bernoulli model. This should fix that.

Reply:
Ok yeah I've noticed that too.